srcpkgs/xdg-desktop-portal-kde/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'xdg-desktop-portal-kde'
 pkgname=xdg-desktop-portal-kde
-version=5.24.2
+version=5.24.3
```

Comment:
Do we really need to update this? I wouldn't touch KDE packages in a GNOME 42 PR.

Reply:
Yeah, I'll remove it.